### DISCUSSION TITLE
Using Screen when converting from CSS_PATH to DEFAULT_CSS for App

### DISCUSSION THREAD

**Discussion by davetapley** [Textualize/textual#2541] [Q&A] [answered]
This _might_ be a bug.

I have a `main.css`:

```css
Screen {
  layout: grid;
  grid-size: 2;
  grid-columns: 1fr 2fr;
}
```

It uses `Screen` per [the docs](https://textual.textualize.io/guide/layout/#vertical):
> `Screen` is the parent container of the widgets yielded from the `App.compose` method, and can be thought of as the terminal window itself.

I'd like to inline it with `DEFAULT_CSS` (to avoid silliness with PyInstaller's `--add-data`).
My _assumption_ is that I should be able to just copy `main.css` verbatim into the `.py` (replacing `CSS_PATH` to `DEFAULT_CSS`).

However that doesn't work, no error, the CSS just isn't applied (no `grid`) ⚠️ 

User error, or bug? 🤔

**Comment by TomJGooding** [comment]
Sorry but just to confirm: you're not trying to use `DEFAULT_CSS` in your app rather than `CSS`?

https://textual.textualize.io/guide/app/#classvar-css

**Comment by willmcgugan** [comment, marked as answer]
`DEFAULT_CSS` has a lower specificity, so that external CSS always wins when rules clash.

If you are moving external stylesheets inline, you should probably use the classvar `CSS` on your App, which has the same specificity as external files.

**Reply by TomJGooding** [reply]
:point_up: Obviously the better answer straight from the horse's mouth - sorry we must have posted almost the exact same time!
